requirements/edx/base.txt

```diff
@@ -41,7 +41,7 @@ git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.1.6#egg=xblock-d
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
 -e git+https://github.com/raccoongang/xblock-pdf.git@sgab-v.1.0.1#egg=xblock-pdf
--e git+https://github.com/edly-io/xblock-video.git@dev#egg=video_xblock
```

Comment:
Why are we removing `video_xblock`?

Reply:
@imhassantariq  removed duplicate insertion, the line `22` states the same entry.